test/snapshots/test.js.md

```diff
@@ -99,12 +101,20 @@ Generated by [AVA](https://ava.li).
         "name": "output",␊
         "plugins": ["vue"]␊
       },␊
+      "eslintConfig": {␊
+        "extends": [␊
+          "eslint:recommended",␊
```

Comment:
no need to enforce code style with `eslint:recommended`, let's be less opinionated. 🤔